src/MongoDB/Driver/Cursor.cpp

```diff
@@ -89,74 +102,31 @@ Array HHVM_METHOD(MongoDBDriverCursor, __debugInfo)
 	Array retval = Array::Create();
 
-	if (data->cursor) {
-		Array cretval = Array::Create();
-		const bson_t *tags = mongoc_read_prefs_get_tags(data->cursor->read_prefs);
-
-		cretval.add(s_MongoDBDriverCursor_stamp, (int64_t) data->cursor->stamp);
-		cretval.add(s_MongoDBDriverCursor_is_command, !!(data->cursor->is_command));
-		cretval.add(s_MongoDBDriverCursor_sent, !!data->cursor->sent);
-		cretval.add(s_MongoDBDriverCursor_done, !!data->cursor->done);
-		cretval.add(s_MongoDBDriverCursor_end_of_event, !!data->cursor->end_of_event);
-		cretval.add(s_MongoDBDriverCursor_in_exhaust, !!data->cursor->in_exhaust);
-		cretval.add(s_MongoDBDriverCursor_has_fields, !!data->cursor->has_fields);
-
-		{
-			Variant v_query;
-
-			hippo_bson_conversion_options_t options = HIPPO_TYPEMAP_INITIALIZER;
-			BsonToVariantConverter convertor(bson_get_data(&data->cursor->query), data->cursor->query.len, options);
-			convertor.convert(&v_query);
-			cretval.add(s_MongoDBDriverCursor_query, v_query);
-		}
-		{
-			Variant v_fields;
-
-			hippo_bson_conversion_options_t options = HIPPO_TYPEMAP_INITIALIZER;
-			BsonToVariantConverter convertor(bson_get_data(&data->cursor->fields), data->cursor->fields.len, options);
-			convertor.convert(&v_fields);
-			cretval.add(s_MongoDBDriverCursor_fields, v_fields);
-		}
-		if (tags->len) {
-			Array rp_retval = Array::Create();
-			Variant v_read_preference;
-
-			rp_retval.add(s_MongoDBDriverCursor_read_preference_mode, (int64_t) mongoc_read_prefs_get_mode(data->cursor->read_prefs));
-
-			hippo_bson_conversion_options_t options = HIPPO_TYPEMAP_DEBUG_INITIALIZER;
-			BsonToVariantConverter convertor(bson_get_data(tags), tags->len, options);
-			convertor.convert(&v_read_preference);
-			rp_retval.add(s_MongoDBDriverCursor_read_preference_tags, v_read_preference);
-
-			cretval.add(s_MongoDBDriverCursor_read_preference, rp_retval);
-		} else {
-			cretval.add(s_MongoDBDriverCursor_read_preference, Variant());
-		}
-
-		cretval.add(s_MongoDBDriverCursor_flags, (int64_t) data->cursor->flags);
-		cretval.add(s_MongoDBDriverCursor_skip, (int64_t) data->cursor->skip);
-		cretval.add(s_MongoDBDriverCursor_limit, (int64_t) data->cursor->limit);
-		cretval.add(s_MongoDBDriverCursor_count, (int64_t) data->cursor->count);
-		cretval.add(s_MongoDBDriverCursor_batch_size, (int64_t) data->cursor->batch_size);
-
-		cretval.add(s_MongoDBDriverCursor_ns, data->cursor->ns);
-
-		if (data->cursor->current) {
-			Array doc_retval = Array::Create();
-			Variant v_doc;
-
-			hippo_bson_conversion_options_t options = HIPPO_TYPEMAP_INITIALIZER;
-			BsonToVariantConverter convertor(bson_get_data(data->cursor->current), data->cursor->current->len, options);
-			convertor.convert(&v_doc);
-			cretval.add(s_MongoDBDriverCursor_current_doc, v_doc);
-		}
-
-		retval.add(s_MongoDBDriverCursor_cursor, cretval);
+	if (data->m_db) {
+		retval.add(s_MongoDBDriverCursor_database, data->m_db);
+	}
```

Comment:
Are you missing an `} else { retval.add(s_MongoDBDriverCursor_database, Variant()); }` here?